internal/cloudsql/instance_test.go

```diff
@@ -74,7 +92,7 @@ func TestConnectInfo(t *testing.T) {
 		t.Fatalf("failed to generate keys: %v", err)
 	}
 
-	im, err := NewInstance(instConnName, client, key, 30*time.Second)
+	im, err := NewInstance(cn.String(), client, key, 30*time.Second)
```

Comment:
I'd like to improve the assertions in all these tests, but consider that work to be part of #9, especially now that we have this mock API.